Application/Setting/Univention/UniventionRole.php

```diff
@@ -0,0 +1,100 @@
+<?php
+namespace SPHERE\Application\Setting\Univention;
+
+use SPHERE\Application\Setting\Univention\Service\Entity\TblUnivention;
+
+class UniventionRole
+{
+    private $curlhandle;
+    private $server;
+    private $token;
+
+    public function __construct() {
+
+        if(($tblUnivention = Univention::useService()->getUnivention(TblUnivention::TYPE_VALUE_SERVER))){
+            $this->server = $tblUnivention->getValue();
+        }
+        if(($tblUnivention = Univention::useService()->getUnivention(TblUnivention::TYPE_VALUE_TOKEN))){
+            $this->token = $tblUnivention->getValue();
+        }
+
+        $this->curlhandle = curl_init();
+    }
+
+    /**
+     * @return array|bool|string
+     */
+    public function getAllRoles()
+    {
+
+        curl_reset($this->curlhandle);
+
+        curl_setopt_array($this->curlhandle, array(
+            CURLOPT_URL => 'https://'.$this->server.'/v1/roles/',
+            CURLOPT_HTTPGET => TRUE,
+            CURLOPT_SSL_VERIFYHOST => FALSE,
+            CURLOPT_HTTPHEADER => array('accept: application/json',
+                'Authorization: bearer '.$this->token),
+            //return the transfer as a string
+            CURLOPT_RETURNTRANSFER => TRUE
```

Comment:
Codacy found an issue: [TRUE, FALSE and NULL must be lowercase; expected "true" but found "TRUE"](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)